scripts/circleci_namespace_gc.py

```diff
@@ -0,0 +1,170 @@
+import argparse
+import json
+import logging
+import subprocess
+import re
+from circleci_base import CircleCIBase
+
+
+class NamespaceGC():
+    def __init__(self, noop):
+        self.circleci = CircleCIBase()
+        self.noop = noop
+
+    def get_active_namespaces(self):
+        active_namespaces = []
+        ns_out = subprocess.check_output(["kubectl", "get", "ns"])
```

Comment:
do we want to filter for active namespaces only?

if delete handles repeat request on terminating namespaces, this is a non issue

Reply:
that's a reasonable fix that we should filter for things being terminated. Right now, the assumption is just that it will run again if something fails.